### ISSUE TITLE
More famous quotes on website

### ISSUE THREAD

**Issue by theScrabi** [TeamNewPipe/website#62]
We have quite some user quotes which is nice, and we should stick to it.

But we should also add quotes from online journals, blog, and famous people on the net.

Like here:

medium.mybridge: "NewPipe: A lightweight Youtube frontend for Android."
https://medium.mybridge.co/38-amazing-android-open-source-apps-java-1a62b7034c40

caschys blog:
https://stadt-bremerhaven.de/newpipe-erlaubt-das-abspielen-von-youtube-videos-wenn-der-bildschirm-aus-ist/

I would also like to feature c't but they only wrote about NewPipe in an artikle you'd have to buy, we might need to ask them weather we could quot this article or not.

What I have here is not much, and it's not good, I know. But whenever you find something/somewhat important on the net reporting about newpipe. Lets consider to quote it/him.

**Comment by TheAssassin** [comment]
@theScrabi whether you need to buy an article or not doesn't matter for citing. The German "Urheberrecht" allows you to cite portions of the article if you follow a set of rules. Therefore, it's possible to quote the essential sentences.

**Comment by TheAssassin** [comment]
@theScrabi you should check out the wiki related issue in the main repo where I compiled a list of references for a potential Wikipedia article. There should be a bunch of usable quotes in those.

**Comment by TobiGr** [comment]
Yes I already wanted to do this, but I don't have enough time for it now. If anyone wants to do it, feel free to do so.
There are some technical things I'd like to do first (CSS parallax, CSS refactor and Jekyll cleanup) to make the code more understandable for others. They have been on my todo list for quite some time and because they'll need a bigger amount of time, I prefer to do these things now. I won't have more time than between the semesters.

**Comment by saderror256** [comment]
Here is my review of it:

*"Stable, Fast, Lightweight, and also supports multiple services as well, I use it to listen to music as it includes a background player"*

**Comment by saderror256** [comment]
Also, if you need more public user reviews, https://alternativeto.net/software/newpipe/reviews/

**Comment by Poolitzer** [comment]
Would you like me to add them to the website next to the existing quotes?

**Comment by TheAssassin** [comment]
You can, if you want to.
